src/sonic-host-services/scripts/hostcfgd

```diff
@@ -109,6 +115,22 @@ class Feature(object):
             raise ValueError('Invalid state rendered for feature {}: {}'.format(self.name, target_state))
         return target_state
 
+    def __safe_eval_bool(self, val):
+        """ Safely evaluate the boolean expression, without raising an exception """
+        try:
+            ret = ast.literal_eval(val)
+        except Exception as e:
+            ret = False
```

Comment:
The default value is recorded in two places, one is here, one in the line: ```self.__safe_eval_bool(feature_cfg.get('has_timer', 'False'))```

Reply:
The idea is, if the user programs something, invalid apart from True or False, this shouldn't fail. 

If the user programs anything other than ''True", the value is set to False